itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/WarehouseInstance.java

```diff
@@ -263,6 +266,21 @@ WarehouseInstance runFailure(String command, int errorCode) throws Throwable {
     }
   }
 
+  WarehouseInstance verifyNotificationID(String dumpLocation) throws Exception {
+    FileSystem fs = new Path(dumpLocation).getFileSystem(hiveConf);
+    long currentNotificationID = getCurrentNotificationEventId().getEventId();
+    Path notifyFilePath = new Path(dumpLocation + "/hive/", "_notification_id");
+    assertTrue(fs.exists(notifyFilePath));
+    InputStream inputStream = fs.open(notifyFilePath);
+
+    BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream));
+    for (String line = reader.readLine(); line != null; line = reader.readLine()) {
```

Comment:
if line == null, assertion will get skipped. We should be sure about the value being present.